test/optimizer/old_optimizer_test.cpp

```diff
@@ -225,10 +232,12 @@ TEST_F(OldOptimizerTests, UpdateDelWithIndexScanTest) {
   txn_manager.CommitTransaction(txn);
   EXPECT_EQ(del_scan_plan_seq->GetPlanNodeType(), PlanNodeType::SEQSCAN);
 
+  LOG_WARN("here3");
   // free the database just created
   txn = txn_manager.BeginTransaction();
   catalog::Catalog::GetInstance()->DropDatabaseWithName(DEFAULT_DB_NAME, txn);
   txn_manager.CommitTransaction(txn);
+  LOG_WARN("here4");
 }
 
 }  // namespace test
```

Comment:
I ran make -j4 check and the following tests are failing:

17 - insert_translator_test (Failed)
21 - query_cache_test (Failed)
26 - value_integrity_test (OTHER_FAULT)
27 - zone_map_scan_test (SEGFAULT)
68 - functions_test (SEGFAULT)
97 - simple_query_test (SEGFAULT)
98 - ssl_test (Failed)
123 - aggregate_groupby_sql_test (SEGFAULT)
124 - aggregate_sql_test (SEGFAULT)
128 - distinct_aggregate_sql_test (OTHER_FAULT)
129 - distinct_sql_test (SEGFAULT)
133 - insert_sql_test (SEGFAULT)
135 - optimizer_sql_test (SEGFAULT)
136 - order_by_sql_test (SEGFAULT)
137 - projection_sql_test (SEGFAULT)
138 - string_functions_sql_test (SEGFAULT)
139 - timestamp_functions_sql_test (SEGFAULT)
143 - update_sql_test (SEGFAULT)
154 - zone_map_test (SEGFAULT)
155 - trigger_test (Failed)

You might want to look into it.

Reply:
You "might" want to look into it? I would say that they **definitely** need to look into :grinning: